spec/ruby/library/datetime/strftime_spec.rb

```diff
@@ -37,6 +37,11 @@
     @time.strftime("%v").should == @time.strftime('%e-%b-%Y')
   end
 
+  it "should be able to show default Logger format" do
+    default_logger_format = "%Y-%m-%dT%H:%M:%S.%6N "
+    @time.strftime(default_logger_format).should == "2001-02-03T04:05:06.000000 "
```

Comment:
`@time` is a DateTime instance but the case we care about here is for `Time` instances, right?
Date/DateTime use the pure-Ruby https://github.com/oracle/truffleruby/blob/master/lib/truffle/date.rb, they don't use java.time and they don't use RubyDateFormatter (which we should probably rename for clarity, and potentially simplify since only `Time` is handled).

Reply:
Done.

Reply:
Fixed.